documentation/src/docs/metrics.md

```diff
@@ -58,15 +97,17 @@ public interface OpStatsLogger {
 }
 ```
 
-- reportSuccessEvent() : Used to track Timer of a successful operation and will record the latency in Nanoseconds in required metric. 
-- reportFailEvent() : Used to track Timer of a failed operation and will record the latency in Nanoseconds in required metric.  
-- reportSuccessValue() : Used to track Histogram of a success value.
-- reportFailValue() : Used to track Histogram of a failed value. 
-- toOpStatsData() :  Used to support JMX exports and inner test.
-- clear : Used to clear stats for this operation.
+- `reportSuccessEvent()`: It is used to track the [Timer](http://metrics.dropwizard.io/3.1.0/manual/core/#timers) of a successful operation and will record the latency in nanoseconds in the required metric.
```

Comment:
Remove `It is` from all of these. The original phrasing sounds better and is consistent with the rest of the doc.

Reply:
ok